### PR TITLE
Add learning MFE to devstack

### PR DESCRIPTION
**Pull request by stvstnfrd**
@abutterworth @davidjoy Any reason we don't yet want to include this?

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -343,6 +343,17 @@ services:
...
       - lms
 
+  frontend-app-learning:
+    extends:
+      file: microfrontend.yml
+      service: microfrontend
+    working_dir: '/edx/app/frontend-app-learning'
+    container_name: edx.devstack.frontend-app-learning
+    ports:
+      - "2000:2000"
```
Comment (ormsbee) [inline docker-compose.yml:396]: @abutterworth, @davidjoy: Do we have any target convention for these port numbers?

**Review by ormsbee — APPROVED** [review]
This seems fine to me, after all the builds pass. It looks like a transient issue with edx-when?

File: docker-compose.yml (hunk 1 above)
Comment (abutterworth) [reply docker-compose.yml:396]: We've been counting up for each new MFE. If I recall correctly we started at @reillz10 's birth year (1992). So I guess that's our convention...

**Review by abutterworth — APPROVED** [review]

**Review by davidjoy — APPROVED** [review]

File: repo.sh [hunk 2]
```diff
@@ -31,6 +31,8 @@ repos=(
     "https://github.com/edx/xqueue.git"
     "https://github.com/edx/edx-analytics-pipeline.git"
     "https://github.com/edx/frontend-app-gradebook.git"
+    "https://github.com/edx/frontend-app-learning.git"
+    "https://github.com/edx/frontend-app-program-console.git"
```
Comment (kdmccormick) [inline repo.sh]: Since this isn't in Juniper (afaik), I think you'll want to put this in the `non_release_repos` and `non_release_ssh_repos` lists.

Comment (stvstnfrd) [reply repo.sh]: Thanks for the tip! Done.

**Pull request merged**